site/gatsby-site/src/components/taxa/Taxonomy.js

```diff
@@ -57,10 +57,6 @@ const Taxonomy = ({ taxonomy, incidentId }) => {
   const canEdit =
     isRole('taxonomy_editor') || isRole('taxonomy_editor_' + taxonomy.namespace.toLowerCase());
 
-  if (!canEdit && taxonomy.classificationsArray.length < 1) {
```

Comment:
Why did you remove this? I think this makes the taxonomy not show up in the event no values are associated with the taxonomy. This is the desired behavior.

Reply:
We are going to have a few of these taxonomies through time and we don't want to show them if they are blank and the user can't fill them in. I would take a re-render over displaying it to all users.

Reply:
Oh got it 👍

Reply:
Updated, refactored it a little bit, the re-render will only happen to authenticated users.